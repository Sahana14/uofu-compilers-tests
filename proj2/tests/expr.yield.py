(yield 3)
(yield 3,)
(yield 3,4,5)
(yield 3,4,5,)
(yield from foo)
