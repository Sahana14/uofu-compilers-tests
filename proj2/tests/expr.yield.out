(Module
 (Expr (Yield (Num 3)))
 (Expr (Yield (Tuple (Num 3))))
 (Expr (Yield (Tuple (Num 3) (Num 4) (Num 5))))
 (Expr (Yield (Tuple (Num 3) (Num 4) (Num 5))))
 (Expr (YieldFrom (Name foo))))
